README.md

```diff
@@ -109,3 +109,17 @@ Convert items in Zotero API JSON format to a [supported export format](https://g
 ```
 $ curl -d @items.json -H 'Content-Type: application/json' 'http://127.0.0.1:1969/export?format=bibtex'
 ```
+
+### Import Translation
+
+Convert items in any [import format](https://www.zotero.org/support/kb/importing_standardized_formats)
+to the Zotero API JSON format:
+
+```
+$ curl -d @data.bib -H 'Content-Type: text/plain' http://127.0.0.1:1969/import
+```
+Instead of letting Zotero decide on a suitable translator for this format you can also indicate the translatorID explicitely:
+```
+$ curl -d @data.bib -H 'Content-Type: text/plain' \
+   http://127.0.0.1:1969/import?translatorID=9cb70025-a888-4a29-a210-93ec52da40d4
+```
```

Comment:
We can remove this last part after https://github.com/zotero/translation-server/issues/93.

Reply:
Dan also just found that `curl -d` will strip newlines before submitting the data (which is a bizarre choice if you ask me) -- `--data-binary` should leave them untouched.

Reply:
Fixed, thanks